source/server/admin/admin.cc

```diff
@@ -493,5 +517,20 @@ void AdminImpl::addListenerToHandler(Network::ConnectionHandler* handler) {
   }
 }
 
+::Envoy::Http::HeaderValidatorStats&
+AdminImpl::getHeaderValidatorStats([[maybe_unused]] Http::Protocol protocol) {
+  switch (protocol) {
+  case Http::Protocol::Http10:
+  case Http::Protocol::Http11:
+    return Http::Http1::CodecStats::atomicGet(http1_codec_stats_, *server_.stats().rootScope());
+  case Http::Protocol::Http3:
+    IS_ENVOY_BUG("HTTP/3 is not supported for admin UI");
```

Comment:
Fall through issue here

Reply:
Yes, UHV is enabled in compile-time-options build. Admin endpoint is used in almost every integration test, and it should also be covered by existing admin tests.

Reply:
Also added checks in the admin_test